apps/spruce/.env-cmdrc.json

```diff
@@ -15,6 +15,16 @@
     "REACT_APP_UI_URL": "https://evergreen-staging.corp.mongodb.com",
     "REACT_APP_HONEYCOMB_BASE_URL": "https://ui.honeycomb.io/mongodb-4b/environments/staging"
   },
+  "corp-secure-staging": {
```

Comment:
Could it work to instead switch over the `staging` and `devStaging` environments to the new URL?